e2e/rbd.go

```diff
@@ -232,6 +235,15 @@ var _ = Describe("RBD", func() {
 			e2elog.Failf("failed to create node secret with error %v", err)
 		}
 		deployVault(f.ClientSet, deployTimeout)
+
+		kernelRelease, err = util.GetKernelVersion()
```

Comment:
don't introduce a mistake, and correct it in the next commit, consider adding the helper function in one commit, and using it here instead (or combine the two in a single commit)

Reply:
I see it this way:
Oh! The mistake was not introduced by this PR, it was part of the resize nbd device test already, I was trying to reorder things to reuse them.

Reply:
I have reordered the things now.